### DISCUSSION TITLE
Azuracast login with Crowdsec?

### DISCUSSION THREAD

**Discussion by RadioHerzblut** [AzuraCast/AzuraCast#7445] [Q&A]
Is there a way to secure the Azuracast login with Crowdsec? Maybe there is a bouncer ?

**Comment by saintcore** [comment]
What I've done in the past was running Azuracast behind haproxy and on the haproxy-instances I was running crowdsec which has a designated bouncer for it. See this https://docs.crowdsec.net/u/bouncers/haproxy/

Maybe also worth to mention: Another option with haproxy would be Coraza or literally any other waf if you don't want to use a cloud-based services as crowdsec. For example this https://coraza.io/connectors/coraza-spoa/

**Comment by RadioHerzblut** [comment]
Thanks for your answer. I will looking.
